WebApp-your-API/B2C/README.md

```diff
@@ -0,0 +1,368 @@
+---
+page_type: sample
+languages:
+  - csharp
+products:
+  - aspnet-core
+  - dotnet-core
+  - azure-active-directory-b2c
+name: Enable your Blazor Server to sign-in users and call Web API with the Microsoft identity platform in Azure AD B2C
+urlFragment: ms-identity-blazor-server
+description: "This sample demonstrates an ASP.NET Core Blazor Server application calling an ASP.NET Core Web API that is secured using Azure AD B2C"
+---
+
+# Enable your Blazor Server to sign-in users and call Web API with the Microsoft identity platform in Azure AD B2C
+
+ 1. [Overview](#overview)
+ 1. [Scenario](#scenario)
+ 1. [Contents](#contents)
+ 1. [Prerequisites](#prerequisites)
+ 1. [Setup](#setup)
+ 1. [Registration](#registration)
+ 1. [Running the sample](#running-the-sample)
+ 1. [Explore the sample](#explore-the-sample)
+ 1. [About the code](#about-the-code)
+ 1. [More information](#more-information)
+ 1. [Community Help and Support](#community-help-and-support)
+ 1. [Contributing](#contributing)
+ 1. [Code of Conduct](#code-of-conduct)
+
+![.NET Core](https://github.com/Azure-Samples/ms-identity-blazor-server/workflows/.NET%20Core/badge.svg)
+
+## Overview
+
+This sample demonstrates an ASP.NET Core Blazor Server application calling an ASP.NET Core Web API that is secured using Azure AD B2C.
+
+## Scenario
+
+1. Secure an ASP.NET Core Web API with the Microsoft identity platform.
```

Comment:
"1. Secure an ASP.NET Core Web API with the Microsoft identity platform." does this belong here?